lumigator/python/mzai/backend/jobs/submission.py

```diff
@@ -25,9 +24,8 @@ class RayJobEntrypoint(ABC):
 
     @property
     def command(self) -> str:
-        """The command ran by the ray job, which is passed a JSON-serialized config."""
-        return f"lm-buddy evaluate huggingface --config '{json.dumps(self.config.args)}'"
-
+        """The command passed to the Ray job, which includes a JSON-serialized config. We currently use lm-buddy as a built-in."""
+        return f"python -m lumigator.python.mzai.backend.lm_buddy evaluate huggingface --config '{json.dumps(self.config.args)}'"
```

Comment:
Note the change here to executing the cli module at the root of lm-buddy locally within the lumigator at this point